app/src/main/java/com/yeonae/chamelezone/view/place/PlaceReviewTabFragment.kt

```diff
@@ -6,37 +6,30 @@ import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Toast
 import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.LinearLayoutManager
+import com.yeonae.chamelezone.Injection
 import com.yeonae.chamelezone.R
-import com.yeonae.chamelezone.data.model.Review
+import com.yeonae.chamelezone.network.model.ReviewResponse
 import com.yeonae.chamelezone.view.mypage.MoreButtonFragment
 import com.yeonae.chamelezone.view.place.adapter.PlaceReviewTabRvAdapter
-import com.yeonae.chamelezone.view.place.presenter.PlaceInfoContract
+import com.yeonae.chamelezone.view.place.presenter.PlaceReviewContract
+import com.yeonae.chamelezone.view.place.presenter.PlaceReviewPresenter
 import com.yeonae.chamelezone.view.review.ReviewCreateActivity
 import com.yeonae.chamelezone.view.review.ReviewImageActivity
+import com.yeonae.chamelezone.view.review.ReviewModifyActivity
 import kotlinx.android.synthetic.main.activity_place_detail.*
 import kotlinx.android.synthetic.main.fragment_place_review_tab.*
 
-class PlaceReviewTabFragment : Fragment() {
-    lateinit var presenter: PlaceInfoContract.Presenter
-    private val PLACE_NUMBER = "placeNumber"
+class PlaceReviewTabFragment : Fragment(), PlaceReviewContract.View {
+    override lateinit var presenter: PlaceReviewContract.Presenter
 
-    private val placeReviewList = arrayListOf(
-        Review("yeonjae22", "어제", "place1", "여기 진짜 분위기 이뻐요"),
-        Review("Lsunae", "이틀전", "place2", "다시 가고 싶은 곳이에요!"),
-        Review("hiyong", "일주일전", "place3", "혼자 가도 좋은거같아요")
-    )
-
-    private var isTouch = false
-    private val images = intArrayOf(
-        R.drawable.img1,
-        R.drawable.img2,
-        R.drawable.img3,
-        R.drawable.img4
-    )
+    override fun placeReview(reviewList: List<ReviewResponse>) {
```

Comment:
```suggestion
    override fun showPlaceReview(reviewList: List<ReviewResponse>) {
```
placeReview 라는 함수의 이름이 동작을 나타내고있지 못합니다. 적절한 네이밍을 해주세요

Reply:
수정했습니다!
5ff23ff